apps/web/src/pages/PoolCandidates/ViewPoolCandidatePage/ViewPoolCandidatePage.tsx

```diff
@@ -45,6 +46,7 @@ import AdminContentWrapper from "~/components/AdminContentWrapper/AdminContentWr
 import ExperienceTreeItems from "~/components/ExperienceTreeItems/ExperienceTreeItems";
 import ExperienceAccordion from "~/components/ExperienceAccordion/ExperienceAccordion";
 
+import PoolStatusTable from "../../Users/UserInformationPage/components/PoolStatusSection";
```

Comment:
Would it be worth moving this to the `components` folder since it is being used on more than one page now?

Reply:
Moved to components with [a357a82](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6819/commits/a357a82f53adb3197cce161cee8ece0f650af969).